### PR TITLE
DDF-2748 updates to change log

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

updates changelog on master branch to include v2.10.2

#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth 

#### Select at least one member from relevant component team(s) from below.

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR.

@coyotesqrl
@jlcsmith
@kcwire
@lessarderic
@pklinef
@shaundmorris

#### How should this be tested?

preview at https://github.com/ricklarsen/ddf/blob/DDF-2748/CHANGES.md#2102

#### What are the relevant tickets?
[DDF-2748](https://codice.atlassian.net/browse/DDF-2748)

#### Checklist:
- [ ] Documentation Updated
- [x] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by ricklarsen** [comment]
build now

File: CHANGES.md [hunk 1]
```diff
@@ -104,7 +104,95 @@ _This is a preview of a pending release and subject to change._
 	</li>
 </ul>
 
+## 2.10.2
+	Release Date: 2017-04-07
```
Comment (shaundmorris) [inline CHANGES.md]: We are releasing after this date, but I think this is close enough :-)

Comment (ricklarsen) [inline CHANGES.md]: I can still fix it. It won't match the version on the 2.10 branch though

**Comment by ricklarsen** [comment]
Regenerated report & updated date.

File: CHANGES.md [hunk 2]
```diff
@@ -1,110 +1,277 @@
 # Changes between Versions
 
+
 ## 2.11.0
 	Release Date: Pending
 _This is a preview of a pending release and subject to change._
```
Comment (emmberk) [inline CHANGES.md]: Nit: "and **is** subject to change."

Comment (ricklarsen) [inline CHANGES.md]: fixed.

**Comment by shaundmorris** [comment]
I don't think Change log updates require build verification. Open to other ideas in the future

**Pull request merged**